modules/cluster-logging-must-gather.adoc

```diff
@@ -0,0 +1,170 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="gathering-cluster-logging-data_{context}"]
+= Gathering cluster logging data
+
+You can use the `cluster-logging-must-gather` tool to gather data that is specifically related to cluster logging and the Cluster Logging Operator in your {product-title} cluster.
```

Comment:
Is there actually a tool (or maybe some component) called `cluster-logging-must-gather`? I couldn't find one from poking around a little, and the command in the procedure is just using `oc adm must-gather`. Is it more just something like, "You can use the `must-gather` tool with additional arguments to gather data..."?

Reply:
Updated wording per feedback because this is built on top of the `must-gather` tool.